src/views/SettingsView.vue

```diff
@@ -1,5 +1,260 @@
+        "
+      >
+        <v-col>
+          <h2 class="title-box-settings mb-3">
+            Segurança e controle dos dados
+          </h2>
+          <p class="text-box-settings">
+            Todos os seus dados que armazenamos (RA, histórico e avaliações)
+            poderão ser excluídos a qualquer momento e você tem controle total
+            sobre eles. Ao desativar sua conta, suas avaliações
+            <b>não serão perdidas</b>.
+          </p>
+        </v-col>
+      </v-row>
+    </v-container>
+
+    <v-dialog v-model="dialog" width="450px">
+      <v-card>
+        <v-card-title class="text-h5"> Excluir conta </v-card-title>
+        <v-card-text
+          >Tem certeza que deseja excluir seu usuário? <br /><br />Caso deseje
+          voltar, tudo estará aqui 😀</v-card-text
+        >
+        <v-card-actions>
+          <v-spacer></v-spacer>
+          <v-btn color="green-darken-1" variant="text" @click="dialog = false">
+            Agora não
+          </v-btn>
+          <v-btn color="error" variant="text" @click="removeAccount()">
+            Excluir conta
+          </v-btn>
+        </v-card-actions>
+      </v-card>
+    </v-dialog>
+  </section>
 </template>
+
+<script setup lang="ts">
+import { ref, computed } from 'vue';
+import { ElMessage } from 'element-plus';
+import Auth from '@/services/Auth';
+import ErrorMessage from '@/helpers/ErrorMessage';
+import AliasInitials from '@/helpers/AliasInitials';
+import { API_URL } from '@/environment';
+import User from '@/services/User';
+import axios from 'axios';
+
+// ------- dados do usuário ---------- //
+const user = computed(() => Auth.user);
+const userLogin = computed(() => {
+  return user.value.value?.email.replace('@aluno.ufabc.edu.br', '');
+});
+
+const userInitials = computed(() =>
+  userLogin.value ? AliasInitials(userLogin.value) : '',
+); // argument type error here
+
+const addGoogleAccount = computed(() => {
+  const apiPath = API_URL.replace('/v1', '');
+  return apiPath + '/connect/google?userId=' + user.value.value?._id;
+});
+const addFacebookAccount = computed(() => {
+  const apiPath = API_URL.replace('/v1', '');
+  return apiPath + '/connect/facebook?userId=' + user.value.value?._id;
+});
+const createdAt = computed(() => {
+  return user.value.value?.createdAt.toLocaleDateString('pt-BR', {
+    timeZone: 'UTC',
+  });
+});
+// ------------------------------------------------------- //
+
+// ------------ Modal ------------------ //
+const dialog = ref(false);
+// ------------------------------------------------ //
+
+// ------- Lógica de remoção do usuário ---------- //
+const loading = ref(false);
+
+async function removeUser(): Promise<void> {
+  try {
+    loading.value = true;
+    const res = await User.delete();
+
+    loading.value = false;
+    if (res.data) {
+      Auth.logOut();
+    }
+  } catch (err) {
+    loading.value = false;
+    if (axios.isAxiosError(err)) {
+      ElMessage({
+        message: ErrorMessage(err),
```

Comment:
```suggestion
        message: "Ocorreu um erro ao tentar excluir sua conta",
```
A resposta de erro tem que ser mais amigável para o usuário final, repassar para ele os códigos de erro das requisições é uma experiência de usuário meio ruim

Reply:
Excelente sugestão, muito boa visão de UX